source/03-components/accordion/_accordion.scss

```diff
@@ -0,0 +1,2 @@
+// @file
```

Comment:
Minor nitpick: we don’t use `// @file` at the beginning of Sass partials anymore.

Reply:
@dcmouyard When did we decide that? It's still in the component generator.

Reply:
@kmonahan Hmm, I don’t remember when we made this change, but I believe it was relatively recently.

Reply:
Removed